project-roles/trusted-committer.md

```diff
@@ -0,0 +1,162 @@
+# Title
+
+Trusted Comitter
+
+# Patlet
+
+Many inner-source projects will find themselves in a situation where
+they consistently receive feedback, features, and bug-fixes from a contributor.
+In these situations project maintainers seek ways to recognize and reward the
+work of the contributor above and beyond single contributions.
+
+# Context
+
+  - You are the maintainer of a cross-team library, service, or shared resource
+  - You receive regular contributions
+
+# Problem
+
+  - Project maintainers want to find ways to scale their ability to support a
+  project
+  - Project maintainers want to find ways to scale the longevity of the value
+  delivered by a project
+
+# Forces
+
+  - Over the lifecycle of a project the focus of the maintainers may shift away
+  to accomodate changing business priorities
+  - Contributors seek visible artifacts of their contributions demonstrating
+  value
+  - Motivated contributors looking to push further inner-sourced projects
```

Comment:
I don't get this point ...

Reply:
How about:
`Motivated contributors looking to build expertise through inner-sourced projects`

Reply:
Consider moving this to the context

Reply:
The idea is that you have to have some people that are actively making contributions before the _Trusted Committer_ pattern comes into play.  If you have a silent project with no one working on it then this pattern is not for you!